.travis.yml

```diff
@@ -1,17 +1,24 @@
+dist: xenial
+sudo: required
+
 language: python
 cache: pip
 
-services:
-  - rabbitmq
+before_install:
+  - sudo apt install rabbitmq-server
+addons:
+  apt:
+    update: true
 
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - 3.7
 
 install:
-  - pip install .[rmq,dev]
+  - pip install .[dev]
```

Comment:
interesting, I didn't know that [pip is forgiving with non-existing extras](https://travis-ci.org/aiidateam/plumpy/jobs/527856176#L449)